index.bs

```diff
@@ -77,16 +88,16 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The magnetometer's associated {{Sensor}} subclass is the {{Magnetometer}} class.
-
-The Magnetometer has a <a>default sensor</a>, which is the device's main magnetomer sensor.
+The magnetometer has two associated {{Sensor}} subclasses, {{Magnetometer}} and {{RawMagnetometer}}.
```

Comment:
That doesn't answer my question at all. Currently, you sharing a single *latest reading map* between two different sensor interfaces, with entries which have identical keys. That's not going to end well. ;-)